third_party/wabt/src/interp/jit/jit-backend.cc

```diff
@@ -301,6 +367,96 @@ static void emitBinary(sljit_compiler* compiler, Instruction* instr) {
                  args[0].argw, args[1].arg, args[1].argw);
 }
 
+static void emitFloatBinary(sljit_compiler* compiler, Instruction* instr) {
+  Operand* operands = instr->operands();
+  JITArg args[3];
+
+  for (int i = 0; i < 3; ++i) {
+    operandToArg(operands + i, args[i]);
+  }
+
+  sljit_s32 opcode;
+  sljit_s32 mov = SLJIT_CONV_F32_FROM_SW;
```

Comment:
this does a conversion not a reinterpret operation. there is a new operation for that in the jit compiler